src/App.js

```diff
@@ -1,25 +1,31 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Login from "./components/Login/Login";
+import { useDataLayerValue } from "./context/DataLayer";
+import { auth } from "./fire";
+import { useEffect } from "react";
+import Application from "./components/Application/Application";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  const [{ user }, dispatch] = useDataLayerValue();
+  console.log({ user });
+
+  useEffect(() => {
+    auth.onAuthStateChanged((authUser) => {
+      if (authUser) {
+        // user is logged in
+        if (!user) {
+          dispatch({
+            type: "SET_USER",
+            user: authUser.providerData[0],
+          });
+        }
+      } else {
```

Comment:
you can delete this else statement if we're not doing anything with it right now